src/Mesh/babylon.mesh.vertexData.ts

```diff
@@ -8,23 +11,50 @@
         setIndices(indices: IndicesArray, totalVertices: Nullable<number>, updatable?: boolean): void;
     }
 
+    /**
+     * This class contains the various kinds of data on every vertex of a mesh used in determining its shape and appearance 
+     */
     export class VertexData {
+        /**
+         * A linear array of the the x, y, z position of each vertex  [...., x, y, z, .....]
+         */
         public positions: Nullable<FloatArray>;
+        /**
+         * A linear array of the the x, y, z normal vector of each vertex  [...., x, y, z, .....]
+         */
         public normals: Nullable<FloatArray>;
+        /**
+         * A linear array of the the x, y, z tangent vector of each vertex  [...., x, y, z, .....]
+         */
         public tangents: Nullable<FloatArray>;
+        /**
+         * A linear array of the the u,v mapping of the texture image of each vertex  [...., u, v, .....]
+         */

```

Comment:
Meshes can have multiple set of texture coordinates. For instance diffuse texture can be on set 1 and bump on set 5